README.md

```diff
@@ -472,6 +487,163 @@ async function saveCoverageData (codeBlocks) {
 export default runUnitTests
 ```
 
+
+## running linter
+```js
+// runLinter.js
+import fs from 'fs'
+import saveToFile from './saveToFile'
+import standard from 'standard'
+import forEachCodeBlock from './forEachCodeBlock'
+
+export const paddingText = (width, string, padding = ' ') => {
```

Comment:
You can use the [pad-right](https://www.npmjs.com/package/pad-right) module which is highly optimized. In addition, it will make our essay shorter. 😄